src/Data/Product/Properties.agda

```diff
@@ -8,13 +8,14 @@
 
 module Data.Product.Properties where
 
-open import Axiom.UniquenessOfIdentityProofs
-open import Data.Product.Base
+open import Axiom.UniquenessOfIdentityProofs using (UIP; module Decidable⇒UIP)
+open import Data.Product.Base using (_,_; Σ; _×_; map; swap; ∃; ∃₂)
 open import Function.Base using (_∋_; _∘_; id)
 open import Function.Bundles using (_↔_; mk↔ₛ′)
 open import Level using (Level)
 open import Relation.Binary.Definitions using (DecidableEquality)
 open import Relation.Binary.PropositionalEquality
+  using (_≡_; _≗_; subst; cong; cong₂; cong′; refl)
```

Comment:
Ugh! I see why you might want `_≗_` in `Core`!
What's your take on `using` directives mentioning >5 names? (Style-guide again...)

Reply:
I've got a shortly-incoming PR for #2331. It's a wart.

A `using` directive that's longer than a single line starts to be excessive. But that's my threshold, the line, not the number of names. [In my own code, I tend to go full-explicit all the time. I'm not yet fully convinced that that's the right way to do things - so I'm certainly not ready to argue for that for `stdlib`.]